tasks/wait-for-image-task.yaml

```diff
@@ -58,6 +46,34 @@ spec:
         exit 1
       fi
 
+      # Output
       echo -n "${infos[0]}" | tee "$(results.IMAGE_DIGEST.path)"
       echo -n "${infos[1]}" | tee "$(results.GIT_REF.path)"
       echo -n "${infos[2]}" | tee "$(results.GIT_REPO.path)"
+
+      IMAGE_DIGEST="${infos[0]}"
+      echo "Waiting for attestation for ${IMAGE}@${IMAGE_DIGEST} to appear..."
```

Comment:
It's likely work-in-progress, but still.
It seems that `IMAGE` environment variable isn't defined yet. Also, `${IMAGE}@${IMAGE_DIGEST}` repeats way too many times.

You could do
```bash
IMAGE_WITH_DIGEST="$(params.IMAGE)@${infos[0]}"
```
and reuse that in all places.

Reply:
See https://github.com/stackrox/konflux-tasks/pull/64/commits/a45ec1e70ed1f9e9e32b595cf8a56c124181c12d